packages/squiggle-lang/src/rescript/utility/XYShape.res

```diff
@@ -25,6 +39,7 @@ module T = {
   let xTotalRange = (t: t) => maxX(t) -. minX(t)
```

Comment:
what's the argument for this module being called `T`? it seems like an opportunity for a more idiomatic name?

Reply:
There's a convention of using T to describe the main type of the file. So, T means "XYShape", and all the functions in it refer to XYShape functions.